src/main/java/com/sparrowwallet/sparrow/control/CodexKeystoreImportPane.java

```diff
@@ -0,0 +1,252 @@
+        importButton.setDisable(true);
+        try {
+            Keystore keystore = importer.getKeystore(derivation, secretShareProperty.get());
+            EventManager.get().post(new KeystoreImportEvent(keystore));
+        } catch(ImportException e) {
+            String errorMessage = e.getMessage();
+            if(e.getCause() != null && e.getCause().getMessage() != null && !e.getCause().getMessage().isEmpty()) {
+                errorMessage = e.getCause().getMessage();
+            }
+            setError("Import Error", errorMessage);
+            importButton.setDisable(false);
+        }
+    }
+
+    private void createImportButton() {
+        importButton = new SplitMenuButton();
+        importButton.setAlignment(Pos.CENTER_RIGHT);
+        importButton.setText("Import Keystore");
+        importButton.getStyleClass().add("default-button");
+        importButton.setOnAction(event -> {
+            importButton.setDisable(true);
+            importKeystore(getDefaultDerivation());
+        });
+        String[] accounts = new String[]{"Import Default Account #0", "Import Account #1", "Import Account #2", "Import Account #3", "Import Account #4", "Import Account #5", "Import Account #6", "Import Account #7", "Import Account #8", "Import Account #9"};
+        int scriptAccountsLength = ScriptType.P2SH.equals(wallet.getScriptType()) ? 1 : accounts.length;
+        for(int i = 0; i < scriptAccountsLength; i++) {
+            MenuItem item = new MenuItem(accounts[i]);
+            final List<ChildNumber> derivation = wallet.getScriptType().getDefaultDerivation(i);
+            item.setOnAction(event -> {
+                importButton.setDisable(true);
+                importKeystore(derivation);
+            });
+            importButton.getItems().add(item);
+        }
+
+        importButton.managedProperty().bind(importButton.visibleProperty());
+        importButton.setVisible(false);
+    }
+
+    private List<ChildNumber> getDefaultDerivation() {
+        return defaultDerivation == null || defaultDerivation.getDerivation().isEmpty() ? wallet.getScriptType().getDefaultDerivation() : defaultDerivation.getDerivation();
+    }
+
+    private void onInputChange(boolean empty, boolean validChecksum) {
+        if(!empty) {
+            try {
+                importer.getKeystore(ScriptType.P2WPKH.getDefaultDerivation(), secretShareProperty.get());
+                validChecksum = true;
+            } catch(ImportException e) {
+                invalidLabel.setText("Invalid checksum");
+                invalidLabel.setTooltip(null);
+            }
+        }
+
+        calculateButton.setDisable(!validChecksum);
+        validLabel.setVisible(validChecksum);
+        invalidLabel.setVisible(!validChecksum && !empty);
+    }
+
+    private Node getSecretShareEntry() {
+        VBox vBox = new VBox(20);
+        vBox.setPadding(new Insets(10, 30, 10, 30));
+        vBox.setSpacing(10);
```

Comment:
Apologies, I missed this - this line should be removed. It's overriding the spacing set above in `new VBox(20)`.